--- a/src/SMESH/src/DriverGMF/DriverGMF.cxx
+++ b/src/SMESH/src/DriverGMF/DriverGMF.cxx
@@ -55,7 +55,7 @@ namespace DriverGMF
 
   bool isExtensionCorrect( const std::string& fileName )
   {
-    std::string ext  = boost::filesystem::extension(fileName);
+    std::string ext  = (boost::filesystem::path(fileName)).extension().string();
     switch ( ext.size() ) {
     case 5: return ( ext == ".mesh" || ext == ".solb" );
     case 6: return ( ext == ".meshb" );
